app/config/config.yml

```diff
@@ -64,6 +64,10 @@ doctrine:
         auto_generate_proxy_classes: "%kernel.debug%"
         auto_mapping: true
 
+        dql:
```

Comment:
Tak, widziałem to, ale wydaje mi się strasznie naciągane. A co jeśli format daty się zmieni?